fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/reamortization/LoanReAmortizationServiceImpl.java

```diff
@@ -117,6 +131,23 @@ private void reverseReAmortizeTransaction(LoanTransaction reAmortizeTransaction,
                 LoanReAmortizationApiConstants.externalIdParameterName);
         reAmortizeTransaction.reverse(reversalExternalId);
         reAmortizeTransaction.manuallyAdjustedOrReversed();
+        reProcessLoanTransactions(reAmortizeTransaction.getLoan());
+    }
+
+    private void reProcessLoanTransactions(Loan loan) {
```

Comment:
Why do we need to reprocess the transactions? Reamortization is always the latest transaction, no?